dns/dns_updater.go

```diff
@@ -97,7 +97,7 @@ func (u *updater) initELBs() error {
 
 	u.schemeToDNS = make(map[string]dnsDetails)
 	for scheme, lbDetails := range elbs {
-		u.schemeToDNS[scheme] = dnsDetails{dnsName: lbDetails.DNSName, hostedZoneID: lbDetails.HostedZoneID}
+		u.schemeToDNS[scheme] = dnsDetails{dnsName: lbDetails.DNSName + ".", hostedZoneID: lbDetails.HostedZoneID}
```

Comment:
I noticed that the ELBs in the `schemeToDNS` map did not append a `.` to the DNS Name while the ALBs did.  By making this consistent, the logic in what is now `determineManagedRecordSets` (which was actually not working properly for ELBs in my previous version) can remain a simple hash lookup.  I read through the code and I didn't see anywhere that this would cause an issue but if I have missed something please let me know and I will revert this and take care of it in  `determineManagedRecordSets`.